.github/workflows/mo_pytorch.yml

```diff
@@ -0,0 +1,37 @@
+name: PyTorch
+on: [push, pull_request]
+
+jobs:
+  convert:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Intel OpenVINO
+        run: |
+          sudo curl -o GPG-PUB-KEY-INTEL-OPENVINO-2021 https://apt.repos.intel.com/openvino/2021/GPG-PUB-KEY-INTEL-OPENVINO-2021
+          sudo apt-key add GPG-PUB-KEY-INTEL-OPENVINO-2021
+          sudo echo "deb https://apt.repos.intel.com/openvino/2021 all main" | sudo tee - a /etc/apt/sources.list.d/intel-openvino-2021.list
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends intel-openvino-dev-ubuntu18-2021.1.110 python3-setuptools
+
+      - name: Install dependencies
+        run: |
+          git clone --depth 1 https://github.com/openvinotoolkit/testdata
+          python3 -m pip install \
+                 numpy \
+                 defusedxml \
+                 networkx \
+                 test-generator==0.1.1 \
+                 torch \
+                 torchvision
+
+          git clone --depth 1 https://github.com/facebookresearch/detectron2
+          python3 -m pip install -e detectron2
+
+      - name: Run tests
+        run: |
+          source /opt/intel/openvino_2021/bin/setupvars.sh
```

Comment:
can we work here based on `make install`? In this case this module will be aligned with latest version of OpenVINO main repo.

Reply:
Do you mean use MO from OpenVINO master branch or also compile the binaries?

Reply:
Depends on what you need:
1. if MO is enough, we can only use MO from master.
2. IF binaries are needed as well, we can compile first and run your tests after

Reply:
Integrated tests as a part of Azure build so we will use master OpenVINO